src/js/blocksetting123.js

```diff
@@ -37,11 +31,12 @@ function GenerateJitter(TrialPerBlock, MaxJitter) {
   return jitters;
 }
 
+
 // shuffle all colors (even across blocks)
 const colors = jsPsych.randomization.shuffle(all_colors);
 
 //colors for practice block
-let colorP1 = '#ca9161'; // change color choosing so not same as in main blocks?
+let colorP1 = '#0173b2'; // change color choosing so not same as in main blocks?
```

Comment:
To be consistent with the in-line comment, we should probably pick a color that isn't used in any of the main blocks. Any particular reason why the previous color didn't work?